packages/design-system/src/components/dialog.tsx

```diff
@@ -107,16 +208,32 @@ const contentShow = keyframes({
   from: { opacity: 0, transform: "translate(-50%, -48%) scale(0.96)" },
   to: { opacity: 1, transform: "translate(-50%, -50%) scale(1)" },
 });
-const contentStyle = css(floatingPanelStyle, {
-  position: "fixed",
+
+const centeredContent = {
   top: "50%",
   left: "50%",
   transform: "translate(-50%, -50%)",
```

Comment:
transforms can cause count of bug, of wrong position detection to various visual glitches. In case of fixed elements I would suggest to use { left: 0, top: 0,  bottom: 0, right: 0, margin: auto}